app/src/main/java/org/oppia/android/app/devoptions/mathexpressionparser/MathExpressionParserViewModel.kt

```diff
@@ -33,10 +33,6 @@ class MathExpressionParserViewModel @Inject constructor(
   private val htmlParser by lazy {
     // TODO(#4206): Replace this with the variant that doesn't require GCS properties.
     htmlParserFactory.create(
-      gcsResourceName = "",
```

Comment:
Remove the TODO statement above because it will prevent the issue from being automatically closed.